x-pack/legacy/plugins/index_management/public/components/mappings_editor/components/fields_tree.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { Tree, TreeItem } from './tree';
+
+interface Props {
+  fields: TreeItem[];
+}
+
+export const FieldsTree = ({ fields }: Props) => (
+  <div className="euiCodeBlock euiCodeBlock--fontSmall euiCodeBlock--paddingLarge">
```

Comment:
The `<EuiCode />` component expects a string to be provided as `children` otherwise an error was thrown. Thanks for pointing it, I will add a comment in the code explaining that.